.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,36 +1,58 @@
-name: Bug Report
+name: 🐞 Bug Report
 description: File a bug report
-labels: bug
+labels: ["bug"]
 body:
   - type: markdown
     attributes:
       value: |
         **Thank you for using Orca Slicer and wanting to report a bug.**
+        
+        Please note that this is not the place to make feature requests or ask for help.
```

Comment:
Better to ask users to submit feature requests in GH instead of Discord.
Can we change message here?

Reply:
Changed, better? :)